src/chrome-launcher.ts

```diff
@@ -314,9 +317,10 @@ class Launcher {
   }
 
   kill() {
-    return new Promise(resolve => {
+    return new Promise((resolve, reject) => {
       if (this.chrome) {
         this.chrome.on('close', () => {
+          delete this.chrome;
           this.destroyTmp().then(resolve);
```

Comment:
Hmm can you explain why this line moved? Was there a race in waiting for chrome to close?